Examples/Benchmarks/VectorBenchmark.som

```diff
@@ -0,0 +1,26 @@
+"A benchmark for the Builtin Vector class"
+
+VectorBenchmark = Benchmark (
+
+    benchmark = (
+
+        "Create an Vector and append 1000 items to it"
+        | arr resultVec |
```

Comment:
Maybe it's worth renaming `arr` to `vec` just to avoid confusion over whether `arr` is an `Array` or `Vector`.